### DISCUSSION TITLE
error when making changes to the "Question Answer" section

### DISCUSSION THREAD

**Discussion by ukraine-sdg** [open-sdg/open-sdg#1931] [Q&A]
Hello. The link http://ukrstat.gov.ua/csr_prezent/2020/ukr/st_rozv/metadata/metadata.htm is not displayed in the product, but the link works on the intermediate site. When trying to make changes - an error message. Please tell me how to fix the error.

**Comment by otis-bath** [comment]
@ukraine-sdg This seems to be an issue with the latest html proofer and using mixed https and http links on the site - you could pin to an older version if this wasn't occuring before, I have shown the change in this PR: https://github.com/ukraine-sdg/site/pull/772
